<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DodawanieTerminu</class>
 <widget class="QDialog" name="DodawanieTerminu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="data_od_label">
      <property name="text">
       <string>Data od</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="data_od_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="data_do_label">
      <property name="text">
       <string>Data do</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="data_do_edit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="opis_label">
      <property name="text">
       <string>Opis</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="opis_edit"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="zatwierdz_przycisk">
      <property name="text">
       <string>Zatwierdź</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="anuluj_przycisk">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="blad_label">
      <property name="text">
       <string>Błąd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
